dist/step_definitions/elements.js

```diff
@@ -288,10 +298,12 @@ function _asyncToGenerator(fn) { return function () { var gen = fn.apply(this, a
   Then(/^every "([^"]*)" element should have the same value for element "([^"]*)" attribute "([^"]*)"$/, function (containerName, elementName, attributeName) {
     const self = this;
 
-    return this.currentPage[containerName].first().element(self.currentPage[elementName].locator()).getAttribute(self.currentPage[attributeName + 'Attribute']).then(function (firstElementAttributeValue) {
-      return self.currentPage[containerName].each(function (containerElement) {
-        containerElement.element(self.currentPage[elementName].locator()).getAttribute(self.currentPage[attributeName + 'Attribute']).then(function (attributeValue) {
-          expect(attributeValue).to.be.equal(firstElementAttributeValue);
+    this.currentPage.waitForVisibilityOf(this.currentPage[containerName].first()).then(() => {
```

Comment:
It won't work. Method requires a string not an element ( you'll get an element here).